src/gui/FileBrowser.cpp

```diff
@@ -65,17 +66,51 @@ enum TreeWidgetItemTypes
 
 
 
+void FileBrowser::addContentCheckBox()
+{
+	auto filterWidget = new QWidget(contentParent());
+	filterWidget->setFixedHeight(15);
+	auto filterWidgetLayout = new QHBoxLayout(filterWidget);
+	filterWidgetLayout->setMargin(0);
+	filterWidgetLayout->setSpacing(0);
+
+	auto configCheckBox = [this, &filterWidgetLayout](QCheckBox* box)
+	{
+		box->setAccessibleName("FileBrowserFilterCheckBox");
```

Comment:
Is there a better way to identify the checkboxes for the stylesheet? `setAccessibleName` is designed to provide text for screenreaders, so I don't think it's the right choice here.

Reply:
`QObject` has `setObjectName()` which would be more suitable. In the style sheet you could then write `QCheckBox[objectName="FileBrowserFilterCheckBox"] {...}`. Alternatively, `QCheckBox` could be subclassed, but this seems a bit overkill. What do you think?

Reply:
I have to admit I've never written CSS for Qt. Does the standard descendant combinator work, i.e. `FileBrowser QCheckBox { ... }`? If so, I would go for that, otherwise `objectName` sounds good for now.

Reply:
> 
> 
> I have to admit I've never written CSS for Qt. Does the standard descendant combinator work, i.e. `FileBrowser QCheckBox { ... }`? If so, I would go for that, otherwise `objectName` sounds good for now.

Qt CSS is a near complete equivalent to normal CSS, so the descendant selector works like normal. 👍

Reply:
Thanks @DomClark and @Veratil , I did not know about that. I added another commit, the descendant selector works perfectly fine.